Add tooltips to Record Setup and the Play Setup buttons on the Meter Toolbar

abhi-kr-2100 opened the pull request:
This implements the minor feature requested by #3968.

Signed-off-by: Abhishek Kumar <abhi.kr.2100@gmail.com>

Resolves: https://github.com/audacity/audacity/issues/3968

Added tooltips to the two setup buttons on the Meter Toolbar.

<!-- Use "x" to fill the checkboxes below like [x] -->

- [x] I signed [CLA](https://www.audacityteam.org/cla/)
- [x] The title of the pull request describes an issue it addresses
- [x] If changes are extensive, then there is a sequence of easily reviewable commits
- [x] Each commit's message describes its purpose and effects
- [x] There are no behavior changes unnecessary for the stated purpose of the PR

Recommended:
- [x] Each commit compiles and runs on my machine without known undesirable changes of behavior

vsverchinsky commented on src/toolbars/MeterToolBar.cpp:
```diff
@@ -234,6 +234,7 @@ void MeterToolBar::Populate()
       mRecordSetupButton = safenew AButton(this);
       mRecordSetupButton->SetLabel({});
       mRecordSetupButton->SetName(_("Record Meter"));
+      mRecordSetupButton->SetToolTip(XO("Input Metering"));
```
Could you please change text to `"Record Meter"` and `"Playback Meter"` respectively? They already have a translation

abhi-kr-2100 replied:
Done.

vsverchinsky commented on src/toolbars/MeterToolBar.cpp:
```diff
@@ -234,7 +234,7 @@ void MeterToolBar::Populate()
       mRecordSetupButton->SetLabel({});
       mRecordSetupButton->SetName(_("Record Meter"));
-      mRecordSetupButton->SetToolTip(XO("Input Metering"));
+      mRecordSetupButton->SetToolTip(XO("Record Meter"));
```
Could you please also squash first two commits? That will make commit history clearer

abhi-kr-2100 replied:
Thank you for pointing this out. I've squashed the first two commits into one.

vsverchinsky commented:
Yet one thing, please rebase your changes onto release-3.2.2 branch so that we could include your changes into upcoming release

abhi-kr-2100 commented:
@vsverchinsky I used GitHub desktop app to rebase `fix-issue-3968` onto `release-3.2.2`. I've kept all the incoming changes whenever there were conflicts. There were no conflicts on `MeterToolbar.cpp`; the only file my commits change.

vsverchinsky commented:
You need to drop all commits that came from master branch (via interactive rebase) and force push again, then change target branch of your PR. Alternatively you could create a new branch on top of `release-3.2.2` then cherry pick your commits and create new PR targeting `release-3.2.2`, closing this one.

vsverchinsky reviewed (changes requested):
You need to drop all commits that came from master branch (via interactive rebase) and force push again, then change target branch of your PR. Alternatively you could create a new branch on top of release-3.2.2 then cherry pick your commits and create new PR targeting release-3.2.2, closing this one.

abhi-kr-2100 commented:
@vsverchinsky I did a `git rebase -i HEAD~73` (73 being the total number of commits that was previously being shown). I dropped all except the two which were my own. There was a conflict and I chose to keep my changes. I then force pushed and changed the target of PR to `release-3.2.2`.

abhi-kr-2100 commented:
@vsverchinsky Let's scrap this PR. I'll make a new PR and this time target the `release-3.2.2` branch.
